src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1400,6 +1405,11 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
 
         protected virtual void EnterParameter(ParameterSymbol parameter)
         {
+            if ((parameter.ContainingSymbol as MethodSymbol)?.MethodKind == MethodKind.LocalFunction)
```

Comment:
>if ((parameter.ContainingSymbol as MethodSymbol)?.MethodKind == MethodKind.LocalFunction) [](start = 12, length = 89)

I think this code belongs in VisitLocalFunctionStatement, we probably should visit attribute arguments.